onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java

```diff
@@ -65,9 +66,9 @@ protected void onCreate(Bundle savedInstanceState) {
             ObaRegion currentRegion = Application.get().getCurrentRegion();
 
             if (showValidateRegionDialog(currentRegion)) {
+                mRegionId = currentRegion.getId();
```

Comment:
@azizmb9494 Could you please use `Bundle` to pass data to `RegionValidateDialog` fragment? Something like:

```
Bundle bundle = new Bundle();
bundle.putLong(key, regionId);
rvd.setArguments(bundle);
```
I think, we should try to avoid creating unnecessary members.